{
  Exception e=new Exception("Dummy function",new Exception("Dummy function cause"));
  throw new YarnException(e);
}
